yaml/landuse.yaml

```diff
@@ -1116,3 +1132,14 @@ filters:
       <<: *output_properties
       kind: crane
       kind_detail: {col: "crane:type"}
+  - filter:
+      landuse:
+        - harbour
+        - port
+        - shipyard
+        - wharf
+        - quay
+    min_zoom: 16
```

Comment:
15 for port and shipyard, 16 for wharf and quay?

Reply:
Updated, and added tests in 745f2e5.